pkg/shortnames/shortnames_test.go

```diff
@@ -141,6 +151,16 @@ func TestResolve(t *testing.T) {
 		assert.False(t, resolved.PullCandidates[0].record)
 	}
 
+	// Now another run with enforcing resolution to Docker Hub.
+	for _, test := range tests {
+		resolved, err := Resolve(sysResolveToDockerHub, test.name)
+		require.NoError(t, err, "%v", test)
+		require.NotNil(t, resolved)
+		require.Len(t, resolved.PullCandidates, 1)
+		assert.Equal(t, test.dockerHubValue, resolved.PullCandidates[0].Value.String())
+		assert.False(t, resolved.PullCandidates[0].record)
+	}
+
```

Comment:
The above only tests defined aliases AFAICS. The `doesnotexist` and fully-qualified cases below might also be worth expanding (and perhaps worth making table-driven).